### DISCUSSION TITLE
Curated collections

### DISCUSSION THREAD

**Discussion by erlend-sh** [raffomania/linkblocks#37] [Ideas]
Once a rudimentary import is in place, the primary use case I have in mind for linkblocks is curated collections: https://github.com/omnivore-app/omnivore/discussions/1845

In short, I’d like the ability to create curated lists of roughly the same kind as what you’ll find on IMDB:
https://www.imdb.com/list/ls055592025/

One way to do it would be to allow lists based on tag combinations, e.g.:

#### burnout + open-source

1. https://mikemcquaid.com/open-source-maintainers-owe-you-nothing/
2. https://nolanlawson.com/2017/03/05/what-it-feels-like-to-be-an-open-source-maintainer/
3. https://www.wired.com/story/open-source-coders-few-tired/
4. https://writing.jan.io/2017/03/06/sustainable-open-source-the-maintainers-perspective-or-how-i-learned-to-stop-caring-and-love-open-source.html
5. https://medium.com/@mikeal/time-to-leave-a68294ccb2af#.p8ss5xeqz

The key difference from having a bunch of articles with a certain tag is that a Collection has a ranking, with (1) being the top recommendation. That way you’ll have two key data points that can be used to arrive at a global list of the top3/top5/etc. #burnout+#opensource articles.

I’m making this a discussion because there’s still some details to settle about how to design this. My current thinking is that for any group of articles with a certain label, you can optionally designate 3-5 articles of that label with a ranking, whereas the rest would be unranked.

**Comment by raffomania** [comment]
We already support "combining" tags by linking them together in a new tag:

<img width="723" alt="image" src="https://github.com/raffomania/linkblocks/assets/2369330/737e2344-fa15-47a0-9885-53f1f6f08a57">

Inlining the linked items and ordering them by some sort of custom ranking is an interesting feature. I'm all for building a form of ordering, especially since it allows to add text and other blocks to smoothly go from a jumbled list of bookmarks to a coherent story, when that helps to contextualize the bookmarks.

**Reply by erlend-sh** [reply]
> I'm all for building a form of ordering, especially since it allows to add text and other blocks to smoothly go from a jumbled list of bookmarks to a coherent story, when that helps to contextualize the bookmarks.

Exactly. It might even be better to forego the notion of 'ranking' in favor of plain 'ordering' as you put it. So my list of 1,2,3 wouldn't be for the purpose of grading the articles on quality, but rather just to provide a recommendation for the order in which these articles should be read.

So when reading the 'burnout + open-source' list, you could read Erlend's personal list and order, or you could click the 'Aggregate recommendation' tab to see the articles and order that the "links-hive-mind" has arrived at.

An implicit quality/relevance score can still be calculated based on which articles are shared most frequently for a particular tag.

**Comment by erlend-sh** [comment]
Another use case for curated collections: I frequently link to this article as an example of "offering a free service is risky": https://baremetrics.com/blog/freemium-saas-implode

I'm sure there are other articles like it out there though. So what I'd like to do is start a list like 'freemium + startup-lessons' with this one article, just to begin soliciting other suggestions.

**Comment by erlend-sh** [comment]
Having pondered this for a while, I’ve decided that the *ranking* system (distinct from *ordering*, i.e. recommend read/view/listen-order) that most appeals to me in the context of Linkblocks is a [tier list](https://en.wikipedia.org/wiki/Tier_list), as popularized in games:


![](https://cdnportal.mobalytics.gg/production/2023/01/4b182466-smash-ultimate-tier-list-2023.jpg)

![](https://miro.medium.com/v2/resize:fit:1200/1*15qdFQ70OfXhUp96BioZZg.png)

![image](https://github.com/raffomania/linkblocks/assets/583842/f001cf5f-5c08-4419-982e-823ecd234cc2)

There are dedicated apps for these types of rankings, such as https://tiermaker.com/ and https://www.smashtierlist.com/

Prior art on github:
- https://github.com/vSchiaffino/tierly
- https://github.com/BladeBoles/tiermaker-clone
- https://github.com/Ciremun/mtm

What I like about tier lists is that they leave more room for multiple *top favorites*. It’s easier to name your top 3 favorite movies, as opposed to ranking those three movies from best to “worst” which doesn’t feel right.

Unnumbered rankings also avoids the temptation to explicitly score our stuff with a number, as if works of art can be reduced to a mathematical score.

Link sharing is first and foremost about sharing your *favorite* stuff, so listing stuff you *don’t* recommend is a low priority use-case.

Therefore I’m imagining a tier list along these lines:

1. Best stuff
2. Great stuff
3. Good stuff

Here’s an example tier list based on my favorite tv shows (that I’ve recalled thus far):

#### Best stuff

- Halt and catch fire
- Six feet under
- Succession
- Parks and Recreation
- Better Call Saul
- Dag 
- Always sunny in Philadelphia
- The Wire

#### Great stuff

- Rubicon
- Shadow line
- Andor 
- Severance
- Breaking bad
- Scrubs
- Broen
- Rectify
- The Glory
- Leftovers
- Reservoir Dogs
- Shogun

#### Good stuff

- Fringe
- The Bear
- Terriers
- Game of Thrones
- Fallout
- The Killing
- The Fall

- - - 

The result is that we get more substantive, less numerically judgemental scores such as:

#### Succession
- Recommended by 520 linkers
- 70% best, 20% great, 10% good

#### Parks and Recreation
- Recommended by 7400 linkers
- 30% best, 50% great, 20% good
